theme/src/styles/content-elements/footer/_index.scss

```diff
@@ -20,196 +20,263 @@
-// === Footer ==============================================
-// =========================================================
+// =================================
+// === Footer  =====================
+// =================================
 .footer {
-  background: $footer-background;
-  border-top: 15px solid $footer-background;
-  margin-top: 50px;
-}
-
-._start-page + .footer {
-  margin: 0;
-}
-
-.footer p {
-  color: $footer-color;
-  font-weight: 400;
-}
+  background: var.$blacklight;
+  padding: 140px 0 60px;
+  font-family: var.$font-family-sans-serif;
+  font-weight: var.$font-weight-medium;
+  font-size: 14px;
+  color: var.$white;
+  -webkit-font-smoothing: antialiased;
+  text-transform: uppercase;
+
+  & h1,
+  & h2,
+  & h3,
+  & h4,
+  & h5,
+  & h6 {
+    color: var.$white;
+    font-weight: var.$font-weight-bolder;
+    letter-spacing: 0.2px;
+    -webkit-font-smoothing: antialiased;
+  }
 
-.footer h1,
-.footer h2,
-.footer h3,
-.footer h4,
-.footer h5,
-.footer h6 {
-  color: $footer-color;
-  font-weight: 400;
-  letter-spacing: 0.2px;
-  padding-top: 20px;
-}
+  & a {
+    transition: color 0.2s ease;
+    color: var.$white;
 
-.footer h4 {
-  border-bottom: 1px solid lighten($footer-background, 20%);
-  padding-bottom: 4px;
-}
+    &:hover,
+    &:focus {
+      color: var.$gray-500;
+      text-decoration: none;
+    }
+  }
 
-.footer a {
-  transition: color 0.2s ease;
-  color: $footer-link-color;
-  font-weight: 400;
+  @include mixin.media-breakpoint-down(md) {
+    text-align: center;
+  }
 
-  &:hover,
-  &:focus {
-      background-color: transparent;
-      color: $footer-link-hover-color;
+  &__header {
+    font-size: 14px;
+    text-transform: uppercase;
   }
-}
+  // -----------------------------------------------------------------------------
+  // -----------------------------------------------------------------------------
+
+  // =================================
+  // === Footer Middle ===============
+  // =================================
+  &__middle {
+    border-bottom: 1px solid rgba(var.$white, 0.1);
+    margin-bottom: 40px;
+
+    &-column {
+      @include mixin.media-breakpoint-down(md) {
+        margin-bottom: 67px;
+      }
 
-.no-touch .footer a:hover {
-  color: $footer-link-hover-color;
-}
+      @include mixin.media-breakpoint-between(sm, lg) {
+        &:nth-child(1),
+        &:nth-child(2) {
+          margin-bottom: 120px;
+        }
+      }
+    }
 
-// -----------------------------------------------------------------------------
-// -----------------------------------------------------------------------------
+    @include mixin.media-breakpoint-up(lg) {
+      margin-bottom: 50px;
+      padding-bottom: 83px;
+    }
 
-// === Footer Quick Links===========
-// =================================
-.footer {
-  div[class*="frame-type-menu_"] > ul {
-      list-style: none;
-      padding: 0;
-      padding-bottom: 10px;
+    & p {
```

Comment:
better to use class then tag

Reply:
The same here "We need to style RTE text in the footer"

Reply:
![Screenshot 2020-03-23 17 13 52](https://user-images.githubusercontent.com/17538897/77331649-b3738300-6d29-11ea-9ad4-ae8d27c06f36.png)